third_party/LICENSES_BUNDLED.txt

```diff
@@ -0,0 +1,264 @@
+The Pytorch repository and source distributions bundle several libraries that are 
```

Comment:
hmm. if the LICENSES_BUNDLED.txt is generated. should we put this in .gitignore?

Reply:
Sorry, I changed the approach. The file is now part of the repo, and there is a test to make sure regenerating it from the current repo state creates the same file. Updating the PR description appropriately.

Reply:
I like this approach.